include/lsst/jointcal/Jointcal.h

```diff
@@ -21,7 +21,7 @@ namespace jointcal {
         LSST_CONTROL_FIELD(sourceFluxField, std::string, "name of flux field in source catalog");
 
         JointcalControl() :
-            sourceFluxField("base_CircularApertureFlux_7")
+            sourceFluxField("None")
```

Comment:
Hmmm... None doesn't have special meaning, I just wanted to set it to something that was clearly not a valid field. Previously, jointcal was cout<<'ing the above value somewhere further down the line even though I set a different value for sourceFluxField later on. I made this change to ensure it wasn't pulling something else out of the schema. Validate only checks that the field isn't empty, nothing more.

I can't set it to a blank string because validate() will fail during instantiation.

I want to totally remove sourceFluxField in the future: I don't think we actually need it, but tearing it out is not going to be easy.